app-creator/deserializer.js

```diff
@@ -0,0 +1,438 @@
+var mapStyles = require('users/msibrahim/app-creator:map-styles');
+var examples = require('users/msibrahim/app-creator:examples');
```

Comment:
Move this to a `test` folder and turn this script into a module by adding the following after `var app`.
`exports.app = app;`